src/ng/compile.js

```diff
@@ -3871,6 +3871,12 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
         compile: function() {
             return {
               pre: function attrInterpolatePreLinkFn(scope, element, attr) {
+                if (trustedContext === $sce.MEDIA_URL && isConstantInterpolation(interpolateFn)) {
+                  // Constant MEDIA_URL attributes should just be sanitized and not watched.
+                  attr[name] = interpolateFn(scope);
```

Comment:
Do we still need this assignment? I thought to go back to the previous behavior we'd want to do *no* assignments.

I thought we'd just update [this condition](https://github.com/angular/angular.js/blob/76f3bb0e2bc5a781f06546485fdb9dc46ef9a2de/src/ng/interpolate.js#L245) to return early like it previously did...

Reply:
This was the only way to keep all the tests passing. If you change the condition you link to the a load of other ngSrc etc related tests fail. This is due to how url sanitation has moved to the interpolation rather than being done surely in the compiler.

Reply:
Oh :/

And we can't do this `if + return` outside the directive? Before doing `directives.push`...